src/styles/blocks/section2.scss

```diff
@@ -0,0 +1,67 @@
+.section_2 {
+  margin-top: 79px;
+  text-align: justify;
+  width: 166px;
+  height: 81px;
+}
+
+.section_2__stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 5px;
+}
+
+.section_2__stars_star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin: 0;
+  background-position: center;
+}
+
+.section_2__stars_star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.section_2__stars_text {
+  text-align: right;
+  height: 16px;
+  width: 53px;
+  line-height: 16px;
+  font-size: 10px;
+  margin-left: 10px;
```

Comment:
The use of `margin-left` to space elements can be replaced with the use of `gap` in the flex parent container if it's applicable. This can make the layout more consistent and easier to maintain.